helm/operator/templates/cluster-role.yaml

```diff
@@ -164,3 +164,14 @@ rules:
       - get
       - update
       - create
+  - apiGroups:
```

Comment:
These changes need to be added to `resources/base/cluster-role.yaml` as well.

Reply:
added via https://github.com/minio/operator/pull/1681/commits/f049a4c0b47b03ba805ba244886a48bffa048721